build-system/compile/bundles.config.js

```diff
@@ -119,6 +119,7 @@ exports.jsBundles = {
     minifiedDestDir: './dist',
     options: {
       minifiedName: 'compiler.js',
+      extraGlobs: ['src/builtins/**/*.js', 'extensions/amp-fit-text/**/*.js'],
```

Comment:
These might not be necessary since you're providing the externs

Reply:
It needs the source files for bundling, not just for type checking. See the result of `amp dist --noextensions`:

![Screen Shot 2021-08-03 at 12 02 25 PM](https://user-images.githubusercontent.com/4656974/128048356-8207576a-6103-4554-837f-1ce8ebd99e78.png)